src/sentry_tracing.c

```diff
@@ -11,23 +12,53 @@ sentry__span_get_trace_context(sentry_value_t span)
 
     sentry_value_t trace_context = sentry_value_new_object();
 
-#define PLACE_VALUE(Key, Source)                                               \
+#define PLACE_CLONED_VALUE(Key, Source)                                        \
     do {                                                                       \
         sentry_value_t src = sentry_value_get_by_key(Source, Key);             \
         if (!sentry_value_is_null(src)) {                                      \
-            sentry_value_incref(src);                                          \
-            sentry_value_set_by_key(trace_context, Key, src);                  \
+            sentry_value_set_by_key(                                           \
+                trace_context, Key, sentry__value_clone(src));                 \
         }                                                                      \
     } while (0)
 
-    PLACE_VALUE("trace_id", span);
-    PLACE_VALUE("span_id", span);
-    PLACE_VALUE("parent_span_id", span);
-    PLACE_VALUE("op", span);
-    PLACE_VALUE("description", span);
-    PLACE_VALUE("status", span);
+    PLACE_CLONED_VALUE("trace_id", span);
```

Comment:
I don’t think all of these things should be "cloned". Rather just incref. strings are supposed to be immutable.